cypress/unit/store-mutation-createColumnToCategoryMap.cy.js

```diff
@@ -0,0 +1,38 @@
+import { getters, mutations } from "~/store/index-refactor";
+
+var store = {};
+
+describe("createColumnToCategoryMap", () => {
+
+    beforeEach(() => {
+
+        store = {
+
+            columnToCategoryMapping: {},
+
+            dataTable: [
+
+                {
+                    "column1Name": "value1", "column2Name": "value2"
+                },
+                {
+                    "column1Name": "value3", "column2Name": "value4"
+                }
+            ],
+
+            getColumnNames: p_state => ["column1Name", "column2Name"]
+        };
+    });
+
+    it("Returns a list of columns from the data table", () => {
```

Comment:
So far we have made different tests for each getter and mutation. So this test should be moved to its own test file. I think it could make sense to merge all mutations or all getters. But we should discuss and change that in a different PR if we are going to do that.

Reply:
We're not there yet re: merging. As I mentioned in #292 recently, I will be removing this in an incoming commit.